crates/bifrost/src/providers/replicated_loglet/sequencer/appender.rs

```diff
@@ -277,6 +289,8 @@ impl<T: TransportConnect> SequencerAppender<T> {
                     // timed out!
                     // none of the pending tasks has finished in time! we will assume all pending server
                     // are gray listed and try again
+                    tracing::warn!(pending=%pending_servers, "Timeout waiting on store response");
```

Comment:
Warn and error level should be used when the log message indicate an abnormal situation that the system might not recover from. For stores to timeout of even fail (e.g. some nodes in the nodeset are down), it's not a system-error as it's part of normal operation. For those, we should use info/debug/trace depending on how "actionable" these are.

Reply:
makes sense